docs/whats-new.md

```diff
@@ -38,6 +51,28 @@ View State:
 | Block groups       | 1.57M    | 4261ms  | 9202ms      | -53.69%   |
 | Usa Zip Code Layer | 1.38M    | 2938ms  | 6564ms      | -55.24%   |
 
+*Benchmark run on 2017 Macbook Pro, 3,1 GHz Dual-Core Intel Core i5, 16 GB memory, Intel Iris Plus Graphics 650 1536 MB. Resolution 1140x900. Network transfer time is not included*
+
+### Layer improvements
+
+- All layers now have a `loaders` prop to support additional data formats. See the [Data Loading Guide](/docs/developer-guide/loading-data.md) for examples.
+- All layers that use a texture (`BitmapLayer`, `IconLayer`, `SimpleMeshLayer`) now offer consistent support for a variety of input formats. Newly added support for plain object descriptors (e.g. `{data: <Uint8Array>, width, height}`) and `HTMLVideoElement` that continuously updates. A new prop `textureParameters` also allows easy customization of texture settings.
+- `HeatmapLayer` supports new prop `aggregation` which can be either `SUM` (default) or `MEAN`.
+- `BitmapLayer` picking callbacks now provide information on which pixel was picked.
+- `GlobeView` now works with `COORDINATE_SYSTEM.CARTESIAN` and `SunLight`.
+- `TileLayer` now supports `modelMatrix`.
+- `IconLayer` adds `onIconError` callback.
+- `OrthographicView` adds `flipY` option.
+- `BitmapLayer` adds experimental `_imageCoordinateSystem` prop. Raster tiles may render correctly in `GlobeView`.
+- `wrapLongitude` now has more consistent behavior across the core layer catalog.
+
+Additionally, various layers received performance optimization, corrected lighting calculation, and API consistency improvements. See "Upgrade Guide" for a list breaking changes in this release.
+
+### @deck.gl/carto
+
+TODO
```

Comment:
@alasarr

Reply:
Add at https://github.com/visgl/deck.gl/pull/5397/commits/0f566c00c550067d577dac80d211c780c5baa2bc

I've also included the improvements in declarative language